routers/web/repo/actions/view.go

```diff
@@ -28,7 +29,9 @@ import (
 	"code.gitea.io/gitea/modules/util"
 	"code.gitea.io/gitea/modules/web"
 	actions_service "code.gitea.io/gitea/services/actions"
+	"code.gitea.io/gitea/services/cicdfeedback"
 	context_module "code.gitea.io/gitea/services/context"
+	"github.com/6543/cicd_feedback"
```

Comment:
Lint fails here

Reply:
I don't expect the linter to not scream :laughing: 

this pull is for demonstration ... but I thin some code parts need refactoring first anywa ... to make it an nice integration.

e.g. I think the "gitea action" code has some  :spaghetti:  vibes!!! witch we should solve first